src/fuzzer.py

```diff
@@ -136,10 +136,17 @@ def fuzzing_round(self, test_case: TestCase, inputs: List[Input]) -> Optional[Eq
         # at this point we need to increase the effectiveness of inputs
         # so that we can detect contract violations (note that it wasn't necessary
         # up to this point because we weren't testing against a contract)
-        boosted_inputs: List[Input] = self.boost_inputs(inputs, 1)
+        boosted_inputs: List[Input]
+        
+        if CONF.analyser_compute_all_ctraces:
```

Comment:
Would it make sense to move this whole block of code inside `boost_inputs`?

Reply:
Done (moved into `trace_and_boost`).